### DISCUSSION TITLE
count_all_changes reports only credible changes

### DISCUSSION THREAD

**Discussion by VenomEvoDevo** [hahnlab/CAGEE#27] [Q&A] [answered]
Hello,

I've successfully run cagee. However, when I speficy with the flag `count_all_changes`, the `clade_results.tab` still reports only a subset of branches (which I believe are the credible changes).
Also, when I count the number of genes with * in the `change.tab` file, the total does not correspond to the number reported in the `clade_results.tab` for the same node or species. Shouldn't be the same?

Thank you very much for the help.

Regards,

Giulia

**Comment by benfulton** [comment, marked as answer]
Sorry I haven't gotten back to you until now. Your understanding is correct, if you add the increases and decreases together it should be the same as the number of asterisks in `change.tab`, and `clade_results.tab` should show all of the branches. If you're still seeing this issue, you might upgrade your version to the latest if you haven't already. If you have, and you are able to provide your transcripts and tree to us, we can take a closer look.
